### ISSUE TITLE
Add LiveReload middleware

### ISSUE THREAD

**Issue by zeke** [zeke/jus#16]
https://github.com/mklabs/tiny-lr#middleware

```js
var port = process.env.LR_PORT || process.env.PORT || 35729;

var path    = require('path');
var express = require('express');
var tinylr  = require('tiny-lr');
var body    = require('body-parser');

var app = express();

// This binds both express app and tinylr on the same port


app
  .use(body())
  .use(tinylr.middleware({ app: app }))
  .use(express.static(path.resolve('./')))
  .listen(port, function() {
    console.log('listening on %d', port);
  });
```

**Comment by motleydev** [comment]
This is basically an essential. Maybe you would consider using BrowserSync? It has a fantastic set of options including syncing scroll position and form field interaction across different devices. Perhaps this would be a good drop in?

https://www.npmjs.com/package/connect-browser-sync

My current solution is to use the terminal command in an elaborate NPM run script.

**Comment by zeke** [comment]
Hey @motleydev I just added browsersync support in https://github.com/zeke/jus/commit/0241a21c7cad7d5c485d7becde53513d53968561 and published 0.17.0. It's working like a charm for me. Care to give it a go?
